src/main/java/me/jellysquid/mods/sodium/client/util/workarounds/DriverWorkarounds.java

```diff
@@ -13,6 +13,10 @@ public static void beforeContextCreation() {
         if (Workarounds.isWorkaroundEnabled(Workarounds.Reference.NVIDIA_BAD_DRIVER_SETTINGS)) {
             NVIDIAWorkarounds.install();
         }
+
+        if (Workarounds.isWorkaroundEnabled(Workarounds.Reference.NVIDIA_BAD_DRIVER_LINUX)) {
+            NVIDIAWorkarounds.setLinuxDisableEnv();
```

Comment:
Rather than directly call into the Linux code, just have `NVIDIAWorkarounds.install` pick the code path depending on the operating system.